vespamalloc/src/vespamalloc/malloc/datasegment.h

```diff
@@ -42,7 +42,7 @@ class DataSegment
         checkAndLogBigSegment();
     }
     void enableThreadSupport() { _mutex.init(); }
-    static size_t blockId(const void * ptr)       {
+    static uint32_t blockId(const void * ptr)       {
```

Comment:
Should this also have a return type of `BlockIdT`?

Reply:
Yes, fixed